Krivorotova Tatyana/sum.js

```diff
@@ -0,0 +1,6 @@
+/*       sum        */
+function sum()
+{
+    return arguments.length === 0? 0 : arguments[0] + sum.apply(0, Array.prototype.slice.call(arguments, 1));
```

Comment:
Actually, I'm ok with that solution, but maybe better to use here `reduce` instead of recursion?

Reply:
oh, ignore this comment - I didn't see that recursion is the requirement